pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -236,92 +234,135 @@ def apply(self):
                                  checkmount=False)
 
 
-class SpecifyRepoSpoke(EditTUISpoke, SourceSwitchHandler):
+class SpecifyRepoSpoke(NormalTUISpoke, SourceSwitchHandler):
```

Comment:
I guess no. The problem is that `SpecifyRepoSpoke` has one item which when it is picket is asking for user input. This item gives user default value.

It will be change of behavior.

Reply:
Ok.